src/py/flwr/simulation/app.py

```diff
@@ -63,7 +63,7 @@ def start_simulation(  # pylint: disable=too-many-arguments
     client_fn: Callable[[str], Client],
     num_clients: Optional[int] = None,
     clients_ids: Optional[List[str]] = None,
-    client_resources: Optional[Dict[str, int]] = None,
+    client_resources: Optional[Dict[str, Any]] = None,
```

Comment:
What value types are we expecting here? We're generally trying to avoid `Any`.

Reply:
We are expecting a float number to represent the fractional GPUs rather than the original int. I changed it to `float` in the latest version.